penny_university_frontend/src/actions/index.ts

```diff
@@ -53,7 +56,7 @@ const fetchFollowUps = (chatId, nextPageUrl) => ({
   },
 })
 
-const putFollowUp = (url, followUp) => ({
+const putFollowUp = (url: string, followUp: FollowUp) => ({
```

Comment:
Where's a `FollowUp` defined?

Reply:
Because it was a type that would be used throughout the app I put it in a type declaration file so that it is available globally.

The type declaration file with `FollowUp` is at `reducers/types.d.ts`